devel/fortio/http.go

```diff
@@ -59,7 +59,7 @@ func init() {
 
 // Version is the fortio package version (TODO:auto gen/extract).
 const (
-	Version       = "0.2.2"
```

Comment:
Do we want this to be tied to the release ? Can we do like a bookstore where people manually increase when they update it. The release process is already as complicated as it is and I don't want to add artifacts that are not essential to istio usage.

Reply:
I'm fine with the internal version to be hardcoded / manually changing when I make a significant change for now

I do want the images to be built by postsubmit though